test/application_system_test_case.rb

```diff
@@ -1,5 +1,5 @@
 require "test_helper"
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
+  driven_by :selenium, using: :headless_chrome, screen_size: [1400, 1400]
```

Comment:
You can drop `screen_size` option when using `headless`, since it doesn't actually pop a window open.

Reply:
Ahh word, thank you